guides/source/4_2_release_notes.md

```diff
@@ -268,6 +268,11 @@ Please refer to the [Changelog][action-pack] for detailed changes.
 *   Added an option to disable logging of CSRF failures.
     ([Pull Request](https://github.com/rails/rails/pull/14280))
 
+*   When the Rails server is set to serve static assets, gzip assets will now be
+    served if the client supports it and a pre-generated gzip file (.gz) is on disk.
```

Comment:
Is Sprockets generating them by default again? I don't see that in any of the commits https://github.com/rails/sprockets. Also the discussion left with the suggestion that compressions would be opt-in not the default. https://github.com/rails/sprockets/issues/26